### PR TITLE
Update the specs for empty search results

### PR DESCRIPTION
**Pull request by Julievrz**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | We continued to discuss the issue (https://github.com/PrestaShop/PrestaShop/issues/15118) and decided to add another wording for 404 pages. I also updated the translation domain. 

| Fixed ticket? | Related to https://github.com/PrestaShop/PrestaShop/issues/15118

### CODE REVIEW

**Comment by LouiseBonnard** [comment]
Wow, so great you modified the specs, thanks a million, @Julievrz! Looks all fine to me!

PS: avoid using 'fixes' in the description, prefer 'related to' - 'fixes' is interpreted as it will close the issue once merged

**Review by LouiseBonnard — APPROVED** [review]

File: front-office/search.md [hunk 1]
```diff
@@ -21,3 +21,12 @@ If the visitor reaches a page that contains no product, the page should display
 `Stay tuned! More products will be shown here as they are added.` in `Shop.Theme.Catalog`
 
 A search bar is available right below to allow visitors to try another query: `Search our catalog` in `Shop.Theme.Global`.
+
+## 404 pages 
+
+If the visitor reaches an available page, the page should display the following message:
```
Comment (Robin-Fischer-PS) [inline front-office/search.md:27]:
```suggestion
If the visitor reaches an unavailable page, the page should display the following message:
```

Maybe I'm wrong, but shouldn't it be "unavailable page" for the 404 behaviour ?

Comment (Julievrz) [reply front-office/search.md]: You are completely right, thanks!

**Comment by marionf** [comment]
Thank you @Julievrz :)

**Pull request merged**